microservices/site-launch/lambda-functions/step-functions-trigger/index.ts

```diff
@@ -8,9 +8,11 @@ import logger from "../../shared/logger"
 export const stepFunctionsTrigger = async (event: MessageBody) => {
   const { AWS_REGION, AWS_ACCOUNT_NUMBER, NODE_ENV } = process.env
   const stepFunctionsParams =
-    NODE_ENV === "LOCAL_DEV" ? { endpoint: "http://localhost:8083" } : {}
+    NODE_ENV === "LOCAL_DEV" ? { endpoint: "http://localhost:8003" } : {}
   try {
-    const stepFunctions = new AWS.StepFunctions(stepFunctionsParams)
+    const stepFunctions = new AWS.StepFunctions({
+      endpoint: "http://localhost:8083",
```

Comment:
just a sanity check - si this supposed to be 8083 or 8003?

Reply:
will be removed downstream completely as we migrate to aws infra directly